frontend/src/index.tsx

```diff
@@ -3,16 +3,15 @@ import ReactDOM from 'react-dom';
 import App from './App';
 
 import { createStore } from 'redux';
-import { Provider} from 'react-redux';
+import { Provider } from 'react-redux';
 
 // Redux State
-import reducer from "./store/reduxState";
+import reducer from './store/reduxState';
```

Comment:
I hope it's okay, I converted all the " to ' just to be consistent. We can do either, but you should avoid mix and matching.